### ISSUE TITLE
get nan loss when training target model in overlooking style module

### ISSUE THREAD

**Issue by lixi92** [Flashkong/Source-Free-Object-Detection-by-Learning-to-Overlook-Domain-Style#12]
get nan loss when training target model in overlooking style module on custom datasets even set learning rate to a very small number

**Comment by CHIGIRI19** [comment]
I have the same problem, did you deal with it?

**Comment by HeinZingerRyo** [comment]
Hi there. I ran this repo long time ago for my own research. To address the loss=nan issue, I guess it's originally from bbox loss and you may need to check bbox coordinates as it may be out of bounds after +1 or -1. I cannot clearly remember where to modify.

Furthermore, you may need to downgrade to Python 2.7.12 PyTorch 1.0.0 torchvision 0.2.2 as stated in readme. This is deprecated even years ago but I clearly remember that I can't reproduce the results under much higher python and torch version. Good luck to you.
